lib/src/player_view.dart

```diff
@@ -169,3 +192,44 @@ class PlayerViewState extends State<PlayerView> {
           );
   }
 }
+
+class _PictureInPictureAPI implements PictureInPictureAPI {
+  _PictureInPictureAPI(this._playerViewState);
+
+  final PlayerViewState _playerViewState;
+
+  /// Returns whether the [PlayerView] is currently in Picture-in-Picture (PiP)
+  /// mode.
+  @override
+  Future<bool> get isPictureInPicture =>
+      _playerViewState._invokeMethod<bool>(Methods.isPictureInPicture);
+
+  /// Returns if Picture-In-Picture is available.
+  ///
+  /// Picture-In-Picture is available in the following use-cases:
+  /// - on iOS 14.2 and above. (We disabled PiP on iOS 14.0 and 14.1 due to an
+  /// underlying iOS bug)
+  /// - on tvOS 14 and above.
+  /// - if explicitly enabled through
+  /// `PlaybackConfiguration#isPictureInPictureEnabled` (default is disabled)
+  @override
+  Future<bool> get isPictureInPictureAvailable async =>
+      _playerViewState._invokeMethod<bool>(Methods.isPictureInPictureAvailable);
+
+  /// The [PlayerView] enters Picture-In-Picture mode.
+  /// Has no effects if already in Picture-In-Picture.
+  /// - Starting Picture-In-Picture during casting is not supported and will
+  /// result in a no-op.
```

Comment:
I think it looks prettier if we indent here as well to align with `Starting Picture....`
```suggestion
  ///   result in a no-op.
```